pages/common/tmux.md

```diff
@@ -2,7 +2,6 @@
 
 > Tmux is a terminal multiplexer. It lets you switch easily between several programs in one terminal, detach them (they keep running in the background) and reattach them to a different terminal.
 > More information: <https://github.com/tmux/tmux>.
-
 > By default, your tmux prefix is Ctrl-b.  In the examples below, it'll mention {{prefix}}.
```

Comment:
This format in itself is a bit unusual for the pages. Maybe it should be moved above the more information line?

Reply:
I was attempting to use something close to the Vim page:
> Vim (Vi IMproved), a command-line text editor, provides several modes for different kinds of text manipulation.
> Pressing `i` enters edit mode. `<Esc>` goes back to normal mode, which doesn't allow regular text insertion.
> More information: <https://www.vim.org>.

I agree that the more information link should be last.